packages/shared/src/components/cards/SourceButton.tsx

```diff
@@ -32,7 +32,7 @@ export default function SourceButton({
           id: source.id,
           name: source.name,
           image: source.image,
-          permalink: getSourcePermalink(source.id),
+          permalink: getSourcePermalink(source.handle),
```

Comment:
Shouldnt we use squad for the link?
```
export const getSourcePermalink = (id: string): string =>
  `${process.env.NEXT_PUBLIC_WEBAPP_URL}sources/${id}`;

```

Reply:
We should base it on squad or source indeed.

Reply:
Ah, let me refactor this function to accept type as well. Good catch!